rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -301,6 +301,8 @@ cloud_resource_warning_vcpus: 80.0
 cloud_resource_critical_vcpus: 90.0
 cloud_resource_warning_disk_space: 80.0
 cloud_resource_critical_disk_space: 90.0
+cloud_resource_cpu_allocation_ratio: 2.0
```

Comment:
If someone does change `nova_*_allocation_ratio` will they also know to change these variables that are just slightly differently named (e.g., `nova_ram_allocation_ratio` -> `cloud_resource_mem_allocation_ratio`)?

Relatedly, do we really want to force support to set *4* variables so that monitoring does the right thing?